Week05/lesson10/homework/City Changer/js/index.js

```diff
@@ -0,0 +1,30 @@
+var city = document.getElementById("#city-type").value;
+
+function switchBackground(event){
+event.preventDefault();
+document.querySelector("body").className = "city-type";
+
+    if(city === "!==") {
```

Comment:
What you're doing here in line 7 is comparing the value of `city` with the *string* `"!=="`

If you're goal is to check if the value of city is *nothing*, then you want:

```
if (city === "") {
  // do stuff here
}
```